seccompfilter_rules.h

```diff
@@ -0,0 +1,62 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+#ifndef SLIRP4NETNS_SECCOMPFILTER_RULES_H
+#define SLIRP4NETNS_SECCOMPFILTER_RULES_H
```

Comment:
Minor nit: Given that this is included inline in function, the header guard shouldn't be defined here.

Other than that LGTM.
(However, I can't comment on the list of syscalls due to limited knowledge.)

Reply:
> the header guard shouldn't be defined here.

Why?

Reply:
Header guards are usually defined to avoid re-declarations/re-definitions.
In this case the included file is not a "list of" declarations/definitions but a list of actions to perform.
Meaning, in case you'd have another function in which you were to `#include` this, the actions would not be performed.

Though, this is just a minor nit because it is a single-use currently.

Reply:
thanks, updated

Reply:
(And with minor nit I mean really just pedantic because I don't assume this to become relevant ;).)